examples/inverse/plot_vector_mne_solution.py

```diff
@@ -0,0 +1,45 @@
+"""
+==============================
+Plotting the full MNE solution
+==============================
+
+The source space that is used for the inverse computation defines a set of
+dipoles, distributed across the cortex. When visualizing a source estimate, it
+is sometimes useful to show the dipole directions, as well as their estimated
+magnitude.
+"""
+# Author: Marijn van Vliet <w.m.vanvliet@gmail.com>
+#
+# License: BSD (3-clause)
+import mne
+from mne.datasets import sample
+from mne.minimum_norm import read_inverse_operator, apply_inverse
+
+print(__doc__)
+
+data_path = sample.data_path()
+subjects_dir = data_path + '/subjects'
+
+# Read evoked data
+fname_evoked = data_path + '/MEG/sample/sample_audvis-ave.fif'
+evoked = mne.read_evokeds(fname_evoked, condition=0, baseline=(None, 0))
+
+# Read inverse solution
+fname_inv = data_path + '/MEG/sample/sample_audvis-meg-oct-6-meg-inv.fif'
+inv = read_inverse_operator(fname_inv)
+
+# Read the source space
+fname_src = data_path + '/MEG/sample/sample_audvis-meg-oct-6-fwd.fif'
+src = mne.read_source_spaces(fname_src)
+
+# Apply inverse solution, set pick_ori='full' to obtain a VectorSourceEstimate
+# object
+snr = 3.0
+lambda2 = 1.0 / snr ** 2
+s = apply_inverse(evoked, inv, lambda2, 'dSPM', pick_ori='full')
+
+# Use peak getter to move vizualization to the time point of the peak magnitude
+_, peak_time = s.magnitude().get_peak(hemi='lh')
```

Comment:
WDYT about `abs(s).get_peak(...)`? I think you just need to override the `__abs__` method.